02-Advanced-topics/05-GSD-visualization.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>Remove the material color argument since it is not used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/fresnel-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>